### DISCUSSION TITLE
Can't grant sudo mode.

### DISCUSSION THREAD

**Discussion by fluentmoheshwar** [community/community#30292] [Mobile]
Can't grant sudo mode if I use SwiftKey as my keyboard. Only first digit is typing.

**Comment by stevepopovich** [comment]
Sorry, do you have specific feedback about the GitHub Mobile app?

**Reply by fluentmoheshwar** [reply]
I don't know where the issue is coming from. Every other app works well with SwiftKey except GitHub mobile.

**Reply by stevepopovich** [reply]
What is this about granting sudo mode?

**Reply by fluentmoheshwar** [reply]
> What is this about granting sudo mode?

So I try to do some sudo protected task from my laptop. Then the notification comes up. Then I try to type the number shown on the screen. But when I type only first digit appears.

Like if the code is 10
only 1 is typing.

**Reply by stevepopovich** [reply]
Sorry, what notification? Like the 2FA notification? Can you share a screenshot or video

**Reply by fluentmoheshwar** [reply]
Yeah. I will share a video soon.
